components/Buttons/FilterButton.vue

```diff
@@ -9,7 +9,7 @@
         v-bind="attrs"
         v-on="on"
       >
-        {{ $t('filter') }}
+        {{ $tc('filter') }}
```

Comment:
I'm not in favor of this.

`$tc()` is meant to handle plurals and should only be used to do that, imo.

I don't see the issue with using `.toString()` in TypeScript code. Templates won't need it and it doesn't add much.

Overall, this feels very hacky and like an abuse of the vue-i18n APIs.

I wouldn't be in favor of wrapping `$t()` either, mind you (We should avoid wrapping things up for the hell of it).

Reply:
@MrTimscampi What about extending the interface?

```
declare module 'vue/types/vue' {
interface Vue {
$t: (name: string, attrs?: any) => string
```

Reply:
That should be fine for our use. It's a string or a set of key/values, but we should always get the string version with how we use it.

See: https://github.com/kazupon/vue-i18n/blob/v8.x/types/index.d.ts#L18